### PR TITLE
qdirstat 1.9 (new cask)

### PR DESCRIPTION
**Pull request by jesusha123**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by jesusha123** [comment]
Audit error:

```shell
audit for qdirstat: failed
 - GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
qdirstat
  * line 5, col 2: GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
Error: 1 problem in 1 cask detected.
```

The original repo satisfies the requirement (120 forks, 45 watchers, 1.5k stars): https://github.com/shundhammer/qdirstat. My repo only provides pre-compiled binaries. The application author has approved of me creating a new repo to provide macOS builds. Please let us know how to resolve the "not notable enough" requirement. Can you grant an exception?

**Comment by bevanjkay** [comment]
@jesusha123 We **may** be able to waive the notability requirement here because the repository does not host the source code, but is there any publicly available endorsement of your builds from the upstream source author? For example, it doesn't look like they link to your repo from the readme as an endorsed MacOS distribution.

**Comment by jesusha123** [comment]
@bevanjkay , the public endorsement has been added:

https://github.com/shundhammer/qdirstat?tab=readme-ov-file#latest-news

**Review by bevanjkay — DISMISSED** [review]

**Review by razvanazamfirei — APPROVED** [review]

**Comment by razvanazamfirei** [comment]
Thanks so much for your first contribution, @jesusha123!

File: Casks/q/qdirstat.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask "qdirstat" do
+  version "1.9"
+  sha256 "b05637c023ef9781ea1d45663f228151a7793b79f8af8e8765cf72260cfd74f3"
+
+  url "https://github.com/jesusha123/qdirstat-macos/releases/download/#{version}/QDirStat-#{version}.dmg"
+  name "QDirStat"
+  desc "Graphical application to show where your disk space has gone"
```
[the remark this answers is not in the document]
Comment (bevanjkay) [reply Casks/q/qdirstat.rb]: @razvanazamfirei Not a big one, but just for reference, the style guidelines suggest using British English as first option, https://docs.brew.sh/Prose-Style-Guidelines#style-and-usage

[the remark this answers is not in the document]
Comment (razvanazamfirei) [reply Casks/q/qdirstat.rb]: @bevanjkay, fixed! Thanks! :)

**Review by p-linnane — APPROVED** [review]
Thanks!

**Pull request merged**